tests/fw_rules_tests/policies/expected_output/istio-test1-scheme_output.yaml

```diff
@@ -106,9 +94,8 @@
     - app in (skydive)
     connection:
     - Protocol: TCP
-      Ports:
-      - '30'
-      - '50'
+      dst_ports:
+      - ({30,50}, )
```

Comment:
dst_ports should be a yaml list of integers. No parenthesis and no commas

Reply:
done